storybook/src/components/DescriptionList/DescriptionList.stories.tsx

```diff
@@ -54,6 +54,33 @@ export const MultipleDescriptions: Story = {
   },
 }
 
+export const MultipleTerms: Story = {
+  args: {
+    termsWidth: 'md',
+    children: [
+      <DescriptionList.Group key={1}>
+        <DescriptionList.Term key={1}>Achternaam</DescriptionList.Term>
+        <DescriptionList.Description key={2}>
+          De naam die een persoon van zijn of haar ouders krijgt
+        </DescriptionList.Description>
+      </DescriptionList.Group>,
+      <DescriptionList.Group key={2}>
```

Comment:
Grouping related terms and descriptions is a common and valid approach. Too bad that it prevents the list from automatically sizing its terms column because the terms are no longer grid items.

Multiple terms sharing one description may be an edge case. We might accept that people must choose a width for their terms in this situation, if we get it to work completely. Let’s discuss during our front-end refinement meeting.